README.rst

```diff
@@ -73,6 +73,7 @@ What's New?
 in development
 ~~~~~~~~~~~~~~
 
+* Colorize constant values (new), decorators, parameters default values and annotations with new colorizer adapted from epydoc. This enables to link to name locations automatically.
```

Comment:
From this text, it is not clear what is the new functionality or why I should be excited about this change?

It this an internal refactoring ?

Does this provide any new functionality to end users?

Reply:
> From this text, it is not clear what is the new functionality or why I should be excited about this change?

Please advise another formulation for the changelog, I would be glad to change it.

> It this an internal refactoring ?

There a bit of refactoring, please read the main text presenting the pr: https://github.com/twisted/pydoctor/pull/402#issue-670858879

> Does this provide any new functionality to end users?

Example of constant value colorizing: https://pydoctor--402.org.readthedocs.build/en/402/docformat/restructuredtext/restructuredtext_demo.constants.html

Example of decorators colorizing: https://pydoctor--402.org.readthedocs.build/en/402/api/pydoctor.test.test_sphinx.html#test_prepareCache

Reply:
I think that we don't need to include information about refactoring inside the public changelog.

But the changelog might include information that there is now a new command line option `--pyval-repr-maxlines` that allows you to restrict/configure the number of lines for on output. 

-----------

I guess that the changelog can contain multiple items, as this PR add a lot of new features

* the availability of --pyval-repr-maxlines 
* Syntax highlight for constant values
* Embedded documentation links inside the value of a constant